static/js/jira-server-url.js

```diff
@@ -55,14 +56,61 @@ const setErrorMessage = error => {
 	$("#gheServerURL").addClass("has-error");
 };
 
-/**
- * Hides the error messages
- */
 const hideErrorMessage = () => {
```

Comment:
Removed the comment above this because the function name provides the information we need.